console/console-init/ui/src/modules/project-detail/components/GeneralInfo/GeneralInfo.test.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { render } from "@testing-library/react";
+import { MemoryRouter } from "react-router";
+import { GeneralInfo, IInfoTypePlan } from "./GeneralInfo";
+import { kFormatter } from "utils";
+
+describe("General Info", () => {
```

Comment:
follow unit test guidelines as i already explain.
write some expect, without expect test is not complete.

Reply:
expect doesn't required